recipe/build.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# TELEMAC home directory
+export HOMETEL=$SRC_DIR/telemac-mascaret
+# Configuration file
+export SYSTELCFG=$HOMETEL/configs/systel.cfg
+
+# Configure PATH and PYTHONPATH
+export PATH=$HOMETEL/scripts/python3:$PATH
+export PYTHONPATH=$HOMETEL/scripts/python3 #:$PYTHONPATH
+
+rm -rf $HOMETEL/configs/*
+export MUMPSHOME=$PREFIX
+export SCALAPACKHOME=$PREFIX
+export BLACSHOME=$PREFIX
+export METISHOME=$PREFIX/lib
+
+#linux
+if [[ $(uname) == Linux ]]; then
+   cp $SRC_DIR/systel.linux.cfg $HOMETEL/configs/systel.cfg
+   export USETELCFG=gnu.dynamic.debug # generate also for debug mode
+   compile_telemac.py
+   export USETELCFG=gnu.dynamic
+fi
+#OSX
+if [[ $(uname) == Darwin ]]; then
+   cp $SRC_DIR/systel.macos.cfg $HOMETEL/configs/systel.cfg
+   export USETELCFG=gfort-mpich
+fi
+# Set TELEMAC version in systel.cfg
+sed -i "/^modules:/a version:    $TELEMAC_VERSION" "$SYSTELCFG"
+
+# Name of the configuration to use
+export LD_LIBRARY_PATH=$HOMETEL/builds/$USETELCFG/wrap_api/lib:$HOMETEL/builds/$USETELCFG/lib
+
+compile_telemac.py
+
+#1: the config folder
+export CONFIGS=$PREFIX/configs/
+if [ -d "$CONFIGS" -a ! -h "$CONFIGS" ]
+then
+   echo "$CONFIGS already exists"
+else
+   mkdir $CONFIGS
+fi
+
+#2: the builds folder
+export BUILDS=$PREFIX/builds/
+if [ -d "$BUILDS" -a ! -h "$BUILDS" ]
+then
+   echo "$BUILDS already exists"
+else
+   mkdir $BUILDS
+fi
+
+#3 the scripts folder
+export SCRIPTS=$PREFIX/scripts/
+if [ -d "$SCRIPTS" -a ! -h "$SCRIPTS" ]
+then
+   echo "$SCRIPTS already exists"
+else
+   mkdir $SCRIPTS
+fi
+
+#4 the source folder
+export SOURCES=$PREFIX/sources/
+if [ -d "$SOURCES" -a ! -h "$SOURCES" ]
+then
+   echo "$SOURCES already exists"
+else
+   mkdir $SOURCES
+fi
+
+cp -r $HOMETEL/configs/* $PREFIX/configs     #1
+cp -r $HOMETEL/builds/* $PREFIX/builds       #2
+cp -r $HOMETEL/scripts/* $PREFIX/scripts     #3
+cp -r $HOMETEL/sources/* $PREFIX/sources     #4
+cp $SRC_DIR/telemac-debug.sh $PREFIX/scripts/python3/telemac-debug.sh
```

Comment:
Simplify and use a dedicated folder to store telemac files ?

Reply:
Same for doc and examples

Reply:
you mean to put sources/builds/configs/scripts in the same folder? or merge them into one only folder?

I'd like to keep to same architecture as the official git, because most of the python scripts (compile_telemac.py for instance) would rely on it.

Reply:
Like I do in Windows.
I create a `telemac-mascaret` folder in $PREFIX (or %LIBRARY_PREFIX% on windows) and put all files see https://github.com/nicogodet/telemac-mascaret-feedstock/blob/1acfa26c2d214189e04ce2edd9800ad30ab4bb39/recipe/build-telemac.bat#L20-L34